SpringMVCAnnotationOnlineStore/src/main/webapp/WEB-INF/pages/reportRendering.jsp

```diff
@@ -25,31 +28,183 @@
 <link href='http://fonts.googleapis.com/css?family=Montserrat:400,300,700' rel='stylesheet' type='text/css'>
 <link href="http://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css" rel="stylesheet">
  
- <style>
-#stay {
-	   position:absolute;
-	   bottom:0;
-	   width:100%;
-	   height:60px;   /* Height of the footer */
-	   background:#6cf;
-	}
-
-</style> 
 </head>
 <jsp:include page="_menu.jsp" />
 <body>
  
  <main style="padding:100px">
    
- <div class="container">
+ <div class="report-container">
    <div class="page-title">Reports</div>
+   <c:out value = "${param.month1}"/>
  	</br>
    <div class>
  
- 		<c:forEach items="${customers}" var="customer">
+ 	<!--	<c:forEach items="${customers}" var="customer">
  			<p> Customer ID: ${customer.id} </p>
  			<p> Customer Name: ${customer.firstName} ${customer.lastName} </p>
- 		</c:forEach>
+ 		</c:forEach>-->
+ 	  <sql:setDataSource var = "snapshot" driver = "com.mysql.jdbc.Driver"
+         url = "jdbc:mysql://localhost/jewlz"
+         user = "root"  password = "YourStrong!Passw0rd"/>
+ 
+	  <c:if test="${param.type == 'monthlySales'}">
+      <sql:query dataSource = "${snapshot}" var = "result">
+         SELECT DATE_FORMAT(Order_Date, '%M %Y') AS Order_Month,
+         DATE_FORMAT(Order_Date, '%Y%m') num_month,
+         FORMAT(ROUND(SUM(Order_Total), 2), 2) AS Price
+         FROM orders 
+         WHERE Order_Status = 'Complete'
+         AND Order_Date
+         BETWEEN STR_TO_DATE( CONCAT('01 ', ?), '%d %M %Y') AND LAST_DAY(STR_TO_DATE( CONCAT('01 ', ?), '%d %M %Y'))
```

Comment:
@nolan12gage date format changed in #9, shouldn't require string formatting. Can you check against updated DB?

Reply:
This is a conversion for the parameter string to become a date so that it can be compared against the DB field.